### ISSUE TITLE
go get github.com/HewlettPackard/gas isn't producing binary

### ISSUE THREAD

**Issue by binodluitel** [GoASTScanner/gas#153]
Let me know if this is not the proper way to open an issue.
It started about 2-3 days ago, may be related to last commit but not sure but when I do 

```
$ go get github.com/HewlettPackard/gas
$ gas
-bash: gas: command not found
```

$GOPATH/bin is in my $PATH

I do not see the binary in `$GOPATH/bin`

When I try to compile manually doing

```
$ go build -v -o gas .
github.com/HewlettPackard/gas

```

It compiles fine

When I try running it 

```
$ ./gas
-bash: ./gas: Permission denied
$ chmod +x ./gas
$ ./gas
./gas: line 1: syntax error near unexpected token `newline'
./gas: line 1: `!<arch>'
```

Is anyone having this issue? Or is it just me

**Comment by gcmurphy** [comment]
The redesign treats gas as a library, with the executable code living under cmd/gas.

Try this to install: `go get github.com/GoASTScanner/gas/cmd/gas/...`

**Comment by gcmurphy** [comment]
Fixed in #154

**Comment by binodluitel** [comment]
Sweet, thank you very much
